feat: Generate new events with complex arrays

rachel-orti opened the pull request:
This pull request add the generation of events for two new topics:

PRODUCT.RETURNS: events for return requests that include several products
PRODUCT.REVIEWS: events for product reviews

**NB: The JAR with dependencies is currently committed in my branch for testing purposes. It is not intended to be merged.**

dalelane commented on src/main/resources/data/reviews.csv:
```diff
@@ -0,0 +1,101 @@
+Rating,Comment,Size,Length
```
Are 100 reviews enough for the use cases you've got in mind? Do you think we need any more?

rachel-orti replied:
The use case of the tutorial is leveraging the global rating and the size and length rankings. It is not leveraging the comments. So it is not an issue (at least for the use case of the tutorial) if the same comments appear several times.

dalelane replied:
that makes sense, thanks - and we can always add to it in future if we need to illustrate use cases like sentiment analysis that do use the comment text

dalelane reviewed (commented):
I feel sorry for Loosehanger that they're getting products returned every 30 seconds, but otherwise this all looks great :-) 

Please can you remove the jars from the target folder (which I'm sure you were already planning to do) and I'll approve/merge

rachel-orti commented:
> I feel sorry for Loosehanger that they're getting products returned every 30 seconds, but otherwise this all looks great :-)
> 
> Please can you remove the jars from the target folder (which I know you were already planning to do) and I'll approve/merge

I have changed a little bit the default frequency for the return request and product review events :-) 
I have also removed the JARs in the target folder.
